docs/src/main/sphinx/release/release-358.md

```diff
@@ -0,0 +1,40 @@
+# Release 358 (XX May 2021)
+
+## General
+
+* Support any query in {doc}`/sql/show-stats`. ({issue}`8026`)
+* Improve performance of complex queries involving joins and `TABLESAMPLE`. ({issue}`8094`)
+* Improve performance of `ORDER BY ... LIMIT` queries on sorted data. ({issue}`6634`)
```

Comment:
Should we change to Top-N pushdown since that is the name we use in the docs now? https://trino.io/docs/current/optimizer/pushdown.html#top-n-pushdown

Reply:
I guess "sorted data" here refers to connector-sorted. Should this be under "Hive connector" section instead?

Reply:
No, we refer to it as ORDER BY ... LIMIT, as that's what users think about when they type their queries.

Reply:
@mosabua topn pushdown is not involved here (no pushdown into the connector takes place in the case study's query)

@martint i unresolved for my other question

Reply:
> I guess "sorted data" here refers to connector-sorted. Should this be under "Hive connector" section instead?

It's an engine-level improvement. If a connector produces sorted data (and describes its properties to the engine), the engine will be able to take advantage of it. I guess we can add it to both.

Reply:
@mosabua we called it "Top-N pushdown" since we needed a succinct name to refer to that pushdown, and what it means is explained in that documentation section. We don't have a general term "Top-N" defined anywhere.

Reply:
Thanks for clarifying .. I clearly misunderstood.

Reply:
They can observe this with third party connectors, too, if they are running any. That’s why I think it also belongs in the general section. It’s not specific to the Hive connector.